test/benchmark.js

```diff
@@ -0,0 +1,46 @@
+#! /usr/bin/env node
+// -*- js -*-
+
+"use strict";
+
+var createHash = require("crypto").createHash;
+var fork = require("child_process").fork;
+var get = require("https").get;
+var args = process.argv.slice(2);
+if (!args.length) {
+    args.push("-mc", "warnings=false,unsafe,sequences=1000000");
+}
+args.push("--stats");
+var urls = [
+    "https://code.jquery.com/jquery-3.1.1.js",
+    "https://code.angularjs.org/1.6.1/angular.js",
+    "https://cdnjs.cloudflare.com/ajax/libs/mathjs/3.9.0/math.js",
+    "https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.js",
```

Comment:
Yay! Moar test code! Moar failures!

```
https://code.jquery.com/jquery-3.1.1.js
Timing information (compressed 1 files):
- parse: 0.110s
- scope: 0.311s
- squeeze: 0.720s
- mangle: 0.035s
- generate: 0.106s

SHA1: 61302b0a69f3a37216c91240939b029971a2b0ce

https://code.angularjs.org/1.6.1/angular.js
Timing information (compressed 1 files):
- parse: 0.255s
- scope: 0.547s
- squeeze: 1.056s
- mangle: 0.068s
- generate: 0.178s

SHA1: d3da5ad3cc12ad8e78e164adaf8f0cb1483c9943

https://cdnjs.cloudflare.com/ajax/libs/mathjs/3.9.0/math.js
Timing information (compressed 1 files):
- parse: 0.552s
- scope: 1.550s
- squeeze: 17.223s
- mangle: 0.141s
- generate: 0.509s

SHA1: b7f7c28f5c7a1fe4e807c34de983509f8ab9d6b8

https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.js
Timing information (compressed 1 files):
- parse: 0.079s
- scope: 0.125s
- squeeze: 0.201s
- mangle: 0.012s
- generate: 0.040s

SHA1: ac90c5749c1e7b35712c09a39c9fae4dbeae0d22

https://unpkg.com/react@15.3.2/dist/react.js
Timing information (compressed 1 files):
- parse: 0.215s
- scope: 0.607s
- squeeze: 1.117s
- mangle: 0.050s
- generate: 0.168s

SHA1: d4a9bbc27dfcc7131fea648ac7d2132727693d72

http://builds.emberjs.com/tags/v2.11.0/ember.prod.js
Timing information (compressed 1 files):
- parse: 0.500s
- scope: 1.312s
- squeeze: 2.278s
- mangle: 0.143s
- generate: 0.443s

SHA1: 776fb4dcd73859227ed2ab6998b6e18073816c24

https://cdn.jsdelivr.net/lodash/4.17.4/lodash.js
TypeError: Unable to get property 'keep_fnames' of undefined or null reference
   at Anonymous function (Function code:3364:9)
   at Anonymous function (Function code:7962:17)
   at Anonymous function (Function code:5344:13)
   at before (Function code:5327:9)
   at Anonymous function (Function code:2932:28)
   at Anonymous function (Function code:2951:13)
   at doit (Function code:122:13)
   at MAP (Function code:148:48)
   at do_list (Function code:2950:9)
   at Anonymous function (Function code:3049:9)

SHA1: da39a3ee5e6b4b0d3255bfef95601890afd80709

https://cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.js
Timing information (compressed 1 files):
- parse: 0.259s
- scope: 0.819s
- squeeze: 1.553s
- mangle: 0.168s
- generate: 0.270s

SHA1: c8fb5fdfcfbcaaba2832fbbfac3dc94752da7655
```

"Good" news is, `master` also fails the same way.